docs/changelog.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## 0.1.3 (In Progress)
+  * Add support for boot2docker 1.7. This means supporting a boot2docker vm that runs 64 bit tiny core linux
```

Comment:
how about `Added support for boot2docker 1.7+, which uses 64-bit Tiny Core Linux`